cloudevents/extensions/expirytime.md

```diff
@@ -0,0 +1,58 @@
+# Expiry Time Extension
+
+This extension provides a mechanism to hint to [consumers](../spec.md#consumer)
+or [intermediaries](../spec.md#intermediary) a time after which an
+[event](../spec.md#event) may be ignored.
+
+In distributed systems with message delivery guarantees, events may be delivered
+to a consumer some significant amount of time after an event has been sent via
+an intermediary. In this situation, it may be desirable to ignore events that
+are no longer relevant. The [`time` attribute](../spec.md#time) may be used
```

Comment:
s/may/MAY/

Reply:
I went with "could" instead since this sentence isn't supposed to be normative.